.env.example

```diff
@@ -16,3 +16,6 @@ PAGE_FREEZER_API_KEY="API_KEY"
 export WEB_MONITORING_DB_STAGING_URL="https://api-staging.monitoring.envirodatagov.org"
 export WEB_MONITORING_DB_STAGING_EMAIL=""
 export WEB_MONITORING_DB_STAGING_PASSWORD=""
+
+# This is used in diffing_server
+export TORNADO_DEBUG_MODE="False"
```

Comment:
I think this might be better without “Tornado” in the name—maybe `DIFFING_SERVER_DEBUG` or something. I think it would be good in the future (no need to do this now) to have it affect logger levels in other places, too, and a broader name makes that more reasonable to do.